### PR TITLE
Add Proximity Sensor tests and delete outdated tests

### PR DESCRIPTION
**Pull request by xiuqijix**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/23122)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/332152854)

File: generic-sensor/generic-sensor-tests.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ const properties = {
   'AbsoluteOrientationSensor' : ['timestamp', 'quaternion'],
   'RelativeOrientationSensor' : ['timestamp', 'quaternion'],
   'GeolocationSensor' : ['timestamp', 'latitude', 'longitude', 'altitude',
-                         'accuracy', 'altitudeAccuracy', 'heading', 'speed']
+                         'accuracy', 'altitudeAccuracy', 'heading', 'speed'],
+  'ProximitySensor' : ['timestamp', 'max']
```
Comment (Honry) [inline generic-sensor/generic-sensor-tests.js]: Are you missing the `distance` and `near` attribute?

Comment (kaixinjxq) [reply generic-sensor/generic-sensor-tests.js]: If the physical object is outside the sensing range, the attribute `distance` and `near` must return null, it conflicts the common function `assert_reading_not_null`, so I will add tests for them separately.

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**